specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -7252,6 +7657,42 @@
         "resource"
       ]
     },
+    "CassandraViewCreateUpdateParameters": {
+      "description": "Parameters to create and update Cosmos DB Cassandra view.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties to create and update Azure Cosmos DB Cassandra view.",
+          "$ref": "#/definitions/CassandraViewCreateUpdateProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ARMResourceProperties"
+        }
+      ],
+      "required": [
+        "properties"
+      ]
+    },
+    "CassandraViewCreateUpdateProperties": {
+      "description": "Properties to create and update Azure Cosmos DB Cassandra view.",
+      "type": "object",
+      "properties": {
+        "resource": {
+          "description": "The standard JSON format of a Cassandra view",
+          "$ref": "#/definitions/CassandraViewResource"
+        },
+        "options": {
+          "description": "A key-value pair of options to be applied for the request. This corresponds to the headers sent with the request.",
```

Comment:
>This corresponds to the headers sent with the request [](http://example.com/codeflow?start=85&length=53)

We should not send headers in ARM requests. 
Everything should be a part of the request body.

Reply:
this is required as materialized view container will have its own throughput specified during view creation